src/graphnet/models/model.py

```diff
@@ -79,15 +81,32 @@ def fit(
         *,
         max_epochs: int = 10,
         gpus: Optional[Union[List[int], int]] = None,
-        callbacks: Optional[List[Callback]] = None,
+        callbacks: Optional[List[Callback]] = [ProgressBar()],
```

Comment:
It is not quite kosher to add lists as default arguments. (Something about [them being mutable](https://docs.python-guide.org/writing/gotchas/#mutable-default-arguments).) So I would keep the None, and then add the `ProgressBar` to the list of callback if it isn't already there.